.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+notifications:
+  email: false
+language: ruby
+rvm:
+  - 2.1.6
+env:
+  - RAILS_VERSION=4.0.0
```

Comment:
This branch fails on 3.2.19, unless we can conditionally get it to install `protected_attributes` when the Rails (ActiveRecord) version is >=4. 

Seeing as the last time someone touched this gem was in December of last year, I'm not sure if we'll need to update it in the context of Rails 3? Hmm